src/styles/block/stopwatch.scss

```diff
@@ -0,0 +1,66 @@
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 80vmin;
+  height: 80vmin;
+  line-height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  margin: 69px auto;
+
+  &__minute-hand {
+    position: absolute;
+    bottom: 50%;
+    left: 48%;
+    transform-origin: bottom center;
+    transform: rotate(0deg);
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+
+    animation: rotate-minute 600s steps(60) infinite; /* Минутная стрелка за час */
+  }
+
+  &__second-hand {
+    position: absolute;
+    bottom: 50%;
+    left: 49%;
+    transform-origin: bottom center;
+    transform: rotate(0deg);
+    width: 1.5vmin;
+    height: 38vmin;
+    background-color: #2c8000;
+
+    animation: rotate-second 9997ms linear infinite; /* Секундная стрелка за минуту */
```

Comment:
The animation duration for the second hand should be 60 seconds (60000ms) to mimic a real-world clock. The current duration of 9997ms will result in the second hand rotating slightly faster than once per second, which is incorrect for a standard clock.